frontend/fingertips-frontend/components/organisms/BarChartEmbeddedTable/barChartEmbeddedTable.test.tsx

```diff
@@ -171,70 +171,70 @@ describe('BarChartEmbeddedTable', () => {
   };
 
   it('should render BarChartEmbeddedTable component', async () => {
-    await act(() =>
-      render(
-        <BarChartEmbeddedTable healthIndicatorData={mockHealthIndicatorData} />
-      )
+    render(
+      <BarChartEmbeddedTable healthIndicatorData={mockHealthIndicatorData} />
     );
-    expect(screen.getByRole('table')).toBeInTheDocument();
+
+    expect(await screen.findByRole('table')).toBeInTheDocument();
     expect(
-      screen.getByTestId('barChartEmbeddedTable-component')
+      await screen.findByTestId('barChartEmbeddedTable-component')
     ).toBeInTheDocument();
   });
 
   it('should always display benchmark data in the first table row of data', async () => {
-    await act(() =>
+    await act(() => {
       render(
         <BarChartEmbeddedTable
           healthIndicatorData={mockHealthIndicatorData}
           benchmarkData={mockBenchmarkData}
         />
-      )
-    );
+      );
+    });
+
     expect(screen.getAllByRole('row')[2]).toHaveTextContent('England');
     expect(screen.getByTestId('table-row-benchmark')).toBeInTheDocument();
   });
 
   it('should display group data in the second row of table data, when the group selected is not England', async () => {
-    await act(() =>
+    await act(() => {
       render(
         <BarChartEmbeddedTable
           healthIndicatorData={mockHealthIndicatorData}
           benchmarkData={mockBenchmarkData}
           groupIndicatorData={mockGroupData}
         />
-      )
-    );
+      );
```

Comment:
Can we remove the act here too and use the findBy alternatives underneath in the selectors.